app/models/project.rb

```diff
@@ -17,6 +17,6 @@ def description
   end
 
   def score
-    rand(100).round(2)
+    (rand * 100.0).round(2)
```

Comment:
So, turns out between `2.4.3` and `2.5.0`, the behaviour of round has changed - Integers now remain integers even after rounding, and `rand` returns an integer, even if given `100.0` as a float. I couldn't find any changelog or commit info on this, but it seems reasonable, so probably not a bug in ruby :)